data/sheragi/archaeology missions.txt

```diff
@@ -1170,7 +1170,11 @@ mission "Sheragi Archaeology: The Box 7b"
 			`	"I see. I'll have to warn you though, following the footsteps of a doomed species can only get you so far.`
 				goto end
 			label truth
-			`	"A Sheragi black box? That's an incredible find! Be careful about the information it contains, though. They are a doomed species. Don't you humans have a legend about a cursed box? What was it... Pandora's Box?`
+			`	You produce an image of the cube that you found on your holopad. The curator stares at it, and then slowly turns to you with pity in his eyes.`
+			`	"This box is red," he says gently. "I read a thesis once on the prevalence of color blindness in humans, but-"`
+			`	"No, wait!" you cut him off. "That's a human idiom. 'Black box' is what we use to refer to a device designed to collect and preserve flight data in the event of a catastrophic failure."`
```

Comment:
@Amazinite, how about this?
```suggestion
			`	"This box is red," he says gently. "I read a thesis once on the prevalence of color blindness in humans, but - oh! Isn't there a human idiom about a 'black box'?"`
			`	"Yes," reply. "That's a human idiom. 'Black box' is what we use to refer to a device designed to collect and preserve flight data in the event of a catastrophic failure."`
```

Reply:
I feel like this is even further in the direction of the player being too exaggerative.

Reply:
What do you mean?
(Also I just removed the `"That's a human idiom."` that you say.)